src/policy/fees.cpp

```diff
@@ -599,7 +599,8 @@ CBlockPolicyEstimator::CBlockPolicyEstimator(double min_bucket_feerate)
     : nBestSeenHeight(0), firstRecordedHeight(0), historicalFirst(0), historicalBest(0), trackedTxs(0), untrackedTxs(0)
 {
     assert(0 < min_bucket_feerate);
-    static_assert(BASE_BUCKET_FEERATE > 0, "Base feerate must be nonzero");
```

Comment:
Will `BASE_BUCKET_FEERATE` still be used?

Reply:
BASE_BUCKET_FEERATE defines the bucket alignment, MIN and MAX then define the extremities.